test/features/movement.js

```diff
@@ -0,0 +1,65 @@
+
+Feature('Movements', () => {
+
+  const centre = {
+    name: 'bigone',
+    male_capacity: 999,
+    female_capacity: 750,
+    id: 1,
+    male_cid_name: ['bigone male holding', 'smallone male holding'],
+    female_cid_name: ['bigone female office', 'smallone female holding']
+  };
+  const randomInt = () => Math.floor(Math.random() * 100000) + 1000;
+  const validMovementTimestamp = moment().set({hour: 7, minute: 0, second: 0}).format('DD/MM/YYYY HH:mm:ss');
+  const movement = (direction, cid) => ({
+    Output: [{
+      Location: 'bigone male holding',
+      'MO In/MO Out': direction,
+      'MO Ref.': `${randomInt()}`,
+      'MO Date': validMovementTimestamp,
+      'MO Type': 'Occupancy',
+      'CID Person ID': `${cid || randomInt()}`
+    }]
+  });
+
+  Scenario('for today', () => {
+    Scenario('an update with a movement in should show up as an `expected in` and reduce the availability', function () {
+      before(function () {
```

Comment:
Agreed there's a bunch more scenarios that would be good to add. I intend to add more as I do more of the work to add details to the wallboard.